docs/pages/9 - Contrib Modules.md

```diff
@@ -272,36 +272,240 @@ There's an [example project](https://github.com/lihaoyi/cask/tree/master/example
 
 ### Play Framework
 
-Play framework routes generation support.
+This module adds basic Play framework support to mill: 
+- configures mill for Play Framework default directory layout,
+- integrates the Play framework routes compiler,
+- optionally: integrates the Twirl template engine,
+- optionally: configure mill for single module play applications.
 
- 
-To declare a module that needs to generate Play Framework routes, you must mix-in the 
-`mill.playlib.routesModule` trait when defining your module. 
+There is no specific Play Java support, building a Play Java application will require a bit 
+of customization (mostly adding the proper dependencies).  
+
+#### Using the plugin
+
+There are 2 base modules and 2 helper traits in this plugin, all of which can be found
+ in `mill.playlib`.
+
+The base modules:  
+- `PlayModule` applies the default Play configuration(layout, dependencies, routes compilation, 
+twirl compilation and akka http server)
```

Comment:
We should probably capitalize the project names: Twirl compilation and Akka HTTP server